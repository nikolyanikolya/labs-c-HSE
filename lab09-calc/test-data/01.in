1
.5
.25
1.25
2*3/4
(10+56
(3+4)*5
+5++6++7
+5++6++7)
(100@56*2)
sqrt(8)
