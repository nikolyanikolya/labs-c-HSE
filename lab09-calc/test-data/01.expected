1.000
0.500
0.250
1.250
1.500
Error 5:
  (10+56
  ......^
35.000
18.000
Error 7:
  +5++6++7)
  ........^
Error 5:
  (100@56*2)
  ....^
2.828
